submissions/YuliiaHordiichuk/js-dom/script.js

```diff
@@ -0,0 +1,102 @@
+const data = [
+    {
+        title: 'Exotic',
+        desc: 'Exotics are essentially the shorthaired version of the Persian cat. Like their longhaired cousins, these cats are calm and affectionate, preferring a warm lap or plush cushion to intense activity. They have a moderate activity level and will happily get out some energy with cat toys before heading back to your lap.',
```

Comment:
Avoid shortcuts for properties like this. `description` would be more clear, to my mind